### PR TITLE
Login as user

### PR DESCRIPTION
**Pull request by dftd**
https://github.com/mindstellar/Osclass/pull/232 but without the conflicts.

*I deleted the old repo in the meantime so I couldn't reopen the old PR*

### CODE REVIEW

File: oc-includes/osclass/classes/controller/admin/CAdminUsers.php [hunk 1]
```diff
@@ -645,6 +649,24 @@ public function doModel()
                 osc_add_flash_ok_message($msg, 'admin');
                 $this->redirectTo(osc_admin_base_url(true) . '?page=users&action=ban');
                 break;
+            case ('login'):
```
Comment (navjottomer) [inline oc-includes/osclass/classes/controller/admin/CAdminUsers.php:652]: Change it user-login, make more sense.

File: oc-includes/osclass/classes/controller/admin/CAdminUsers.php [hunk 2]
```diff
@@ -645,6 +649,24 @@ public function doModel()
...
+
+                Session::newInstance()->_set('userId', $aUser['pk_i_id']);
+                Session::newInstance()->_set('userName', $aUser['s_name']);
+                Session::newInstance()->_set('userEmail', $aUser['s_email']);
+                $phone = $aUser['s_phone_mobile'] ?: $aUser['s_phone_land'];
+                Session::newInstance()->_set('userPhone', $phone);
+
+                osc_run_hook('after_login', $user, osc_user_dashboard_url());
+                osc_add_flash_ok_message(sprintf(_m('Logged in as %s successfully'), $aUser['s_name']));
+                $this->redirectTo(osc_apply_filter('correct_login_url_redirect', osc_user_dashboard_url()));
```
Comment (navjottomer) [inline oc-includes/osclass/classes/controller/admin/CAdminUsers.php:668]: No need for an added filter, it will be merged.

Comment (dftd) [reply oc-includes/osclass/classes/controller/admin/CAdminUsers.php:668]: No need to change anything?

**Comment by navjottomer** [comment]
Remove the filter.

**Comment by dftd** [comment]
> Remove the filter.

Ah sorry, sure.

**Pull request merged**